XenAdmin/ConsoleView/VNCTabView.cs

```diff
@@ -768,7 +779,7 @@ private string GuessNativeConsoleLabel(VM source)
                     if (osString != null)
                     {
                         if (osString.Contains("Microsoft"))
-                            label = UseRDP;
+                            label = (Helpers.CreedenceOrGreater(targetHost) && !RDPEnabled) ? enableRDP : UseRDP;

```

Comment:
1) We now have different version numbers on trunk so we can use CreamOrGreater() instead of CreedenceOrGreater()
2) The construction "Helpers.CreedenceOrGreater(host) && !RDPEnabled" occurs several times, and it might be better to extract it into a separate function